### DISCUSSION TITLE
Support for SQlite

### DISCUSSION THREAD

**Discussion by legout** [procrastinate-org/procrastinate#928] [Ideas]
Hi guys,

i am currently researching python task queue frameworks. 
So far, I like huey and procrastinate the most. Both are easy to use, stable and fast. 
Unfortunately, huey is not async compatible, thats is where procrastinate shines.  But I like Hueys SQlite storage. I am a huge fan of SQlite, because of its simplicity. 

Therefore, I wonder, if there are any plans, or if it would be easy/hard to develop a Sqlite connector.

Thanks and regards,
legout

**Comment by ewjoachim** [comment]
No, Procrastinate is tightly coupled with PG. Starting to open it to other DBs would lead to a **LOT** of new kinds of headaches, and likely bring a lot of the things I'm not fond of in other _vegetable-named_ Python task queues.

That said, you're welcome to:
- Create a connector in a different project (I suspect that you will run into multiple problems but YMMV)
- Fork the project and reuse the relevant parts

**Reply by legout** [reply]
Ok. Thanks for your reply. :-)
